core/src/PatternLab/Twig/Template.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace PatternLab\Twig;
+
+use Locale;
+use Twig\Template as BaseTemplate;
+
+abstract class Template extends BaseTemplate
+{
+    private static $isRendering = false;
+
+    public function render(array $context)
+    {
+        $isRendering = self::$isRendering;
+
+        if (!$isRendering) {
+            Locale::setDefault($context['lang'] ?? $context['attributes']['lang'] ?? 'en');
+            self::$isRendering = true;
```

Comment:
This is a bit of a hack, since the `localizeddate()` default locale is `Locale::getDefault`. Using `self::$isRendering` here to avoid nesting problems (since `includes` come through here too).

There doesn't seem to be a suitable place in Pattern Lab or Twig to easily hook it when a pattern/template is rendered; this appeared to be the least invasive approach.